packages/locale/res/translations/de.json

```diff
@@ -2005,23 +2013,23 @@
   ],
   [
     "Date",
-    "Datum"
```

Comment:
hmm @MaKleSoft why would it remove the existing translations?

Reply:
Was just because locally they had been removed. I just brought them back from `main` and re-ran `npm run locale:extract` (no need to run it from the package directory), and it's all fine now.